hudi-common/src/test/java/org/apache/hudi/common/table/read/TestUpdateProcessor.java

```diff
@@ -174,6 +175,13 @@ void testHandleInsertWithPayload(boolean shouldIgnore) {
     }
   }
 
+  private static BufferedRecord<IndexedRecord> getRecordWithSerializableIndexedRecord(String value, HoodieOperation operation) {
+    GenericRecord record = new GenericData.Record(SCHEMA);
+    record.put("key", KEY);
+    record.put("value", value);
+    return new BufferedRecord<>(KEY, 1, new HoodieAvroIndexedRecord(record).getData(), 0, operation);
```

Comment:
Now that the `SerializableIndexedRecord` creation is not package private, can we update this code to directly create an instance instead of going through the `HoodieAvroIndexedRecord`?